.github/workflows/test.yml

```diff
@@ -346,6 +346,12 @@ jobs:
       - name: Check cargo deny advisories/licenses
         if: needs.changes.outputs.dependencies == 'true' || needs.changes.outputs.deny == 'true'
         run: make check-deny
+      - name: Install the 3rd-party license tool
```

Comment:
Usually we have this installation in the bootstrap file. Would it make sense to put this there?

Reply:
I could go either way. It's _only_ needed for this one check, so 🤷🏻

Reply:
True, though the bootstrap scripts are also run as part of `make environment` which installs needed development tools.